project_sale_link/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Sale Link",
+    "summary": """
+        Add sale and sale invoice link in project""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "depends": ["sale_timesheet", "project", "account"],
```

Comment:
@adrienpeiffer Is sale_timesheet dependency really needed ?

Reply:
AS project is added by sale_timesheet, yes